style.css

```diff
@@ -1,3 +1,198 @@
-/**
- * Your CSS goes here.
- */
\ No newline at end of file
+
+*{
```

Comment:
Be careful while using * because it is selecting all the elements in your project.